### DISCUSSION TITLE
play sound when wakework detect

### DISCUSSION THREAD

**Discussion by youkorr** [BigBobbas/ESP32-S3-Box3-Custom-ESPHome#26] [Ideas]
play sound when wakework detected would be interesting to integrate it

**Comment by BigBobbas** [comment]
thats an easy one , I will try and build it in to the next changes i've got to do.

**Comment by youkorr** [comment]
Thank

**Comment by IdiotTryingToFindWay** [comment]
How was your test on that?

**Reply by youkorr** [reply]
I had forgotten thank you IdiotTryingToFindWay very much for this screensaver I also managed to put the % and put the battery level

**Comment by youkorr** [comment]
Hi for my part I had already tried but I found myself with a problem it requires a very short sound or otherwise it disrupts wakeword it hears half when you ask for a request

**Reply by IdiotTryingToFindWay** [reply]
yea i think it would be a pain to have some kind of wait command while the audio plays and when it starts relistening. 
You could probably do it in an automation through HA but that seems cumbersome to get it to work correctly.

**Comment by youkorr** [comment]
exact or you have to give a wait time of 1 second so that wakeword then starts to hear you, this is a clue to look for

**Comment by BigBobbas** [comment]
i have it working pretty good... and you can enable and disable from the device,. I also have timers working... i just need to sort a couple of bugs and then i will post the updates needed. I'm just a bit stuck for time at the moment, with normal life. but i'm really hoping to get that posted in the next couple of days.

**Reply by youkorr** [reply]
Super thank you very much for this beautiful custom

**Comment by BigBobbas** [comment]
https://github.com/BigBobbas/ESP32-S3-Box3-Custom-ESPHome/assets/150487209/c10e7b6a-5a19-4c66-a434-7b4ed41ca2ac

Ignore the screen .. just in middle of tweaking

**Reply by youkorr** [reply]
Ok coolin ?

**Comment by BigBobbas** [comment]
https://github.com/BigBobbas/ESP32-S3-Box3-Custom-ESPHome/assets/150487209/e1aea1bc-5e06-45f0-a6d7-bc5640039e1b

**Comment by youkorr** [comment]
beautiful I love the beep and fast and wakeword arrives afterwards the modification of the parameters and great
How did you manage to ask for one ok coolin mine is ok nabu it's not great

**Reply by BigBobbas** [reply]
i use snowboy HA addon for my HA wakewords, and created a custom wakeword using this link https://snowboy.jolanrensen.nl/

**Reply by youkorr** [reply]
thank you very much I didn't know that

**Comment by BigBobbas** [comment]
https://github.com/BigBobbas/ESP32-S3-Box3-Custom-ESPHome/assets/150487209/792aece7-0272-4b09-8598-9a811eb89750

**Reply by youkorr** [reply]
I think it works very well, what's wrong?

**Reply by BigBobbas** [reply]
it requires some change to the way that pages are displayed to incorporate the timer and the wake sound and also still have screensaver all working together . When i post the update, timer displays will not be available as that requires more work to get it to fit in with this firmware. 
But you can set a timer, and it will notify you and you can tap the screen to stop the sound etc.. which is the important part! ... you can set more than 1 timer .. but at the moment there is no way of knowing which timer has finished... so i need to work on that too

**Reply by youkorr** [reply]
Ok

**Comment by youkorr** [comment]
The timer display will be missing

**Comment by BigBobbas** [comment]
ADDED AS EXPERIMENTAL
[HERE](<https://github.com/BigBobbas/ESP32-S3-Box3-Custom-ESPHome/discussions/37>)
